package.json

```diff
@@ -18,9 +18,7 @@
   "license": "Apache License Version 2.0,",
   "main": "jasmine-es6-promise-matchers.js",
   "directories": {},
-  "dependencies": {
-    "es6-promise": "~2.0.1"
```

Comment:
Do our Karma test depend on this?
https://github.com/bvaughn/jasmine-es6-promise-matchers/blob/dd000811d6dca7b8913f0aaa9421e153cbca775e/karma.conf.js#L8

(It would be in `devDependencies` either way.)

Reply:
Yeah they do but the dependency is also installed by devDependencies so it's fine. Actually npm was raising a warning about the duplicate dependency.

Reply:
d'oh, you're right. ok 😄